### ISSUE TITLE
Fix possible settings bug that reverts from "enabled" to "domain list"

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/optmeowt-browser-extension#127]
Per the the comment from one of our users.

<img width="718" alt="Screen Shot 2020-12-21 at 11 37 52 AM" src="https://user-images.githubusercontent.com/11034003/102799851-0ba2e880-4381-11eb-98d0-0725b7f319d0.png">

**Comment by SebastianZimmeck** [comment]
As @kalicki1 explained in our call today (and later in this issue), that is a feature not a bug. 

Possibly, we want to have a readme FAQ, GitHub Wiki, or GitHub discussions for these type of clarifications. @kalicki1 will come up with a plan.

**Comment by kalicki1** [comment]
It seems that using a Wiki may be our best bet for documenting quirks like this when they come up. I wrote up a basic Wiki with a little bit of information about this issue as well as what @dknopf found in issue #130. I also made mention of an FAQ in the readme to point users in that direction if they find something out of the ordinary. If we decide to keep it, we should update the OptMeowt settings about page as well.
